site/config-overrides.md

```diff
@@ -1,108 +1,108 @@
-# Configuration Overrides
+# Configuration overrides
 
-**PLEASE NOTE**:  This page is a work in progress. 
+**PLEASE NOTE**:  This page is a work in progress.
 
 ---
 # Overview
 
-Use configuration overrides to customize a WebLogic domain home configuration without modifying the domain's actual config.xml or system resource files. For example, you may want to override a JDBC Datasource xml module Username, Password, and URL so that it references a local database. 
+Use configuration overrides (also called _situational configuration_) to customize a WebLogic domain home configuration without modifying the domain's actual `config.xml` or system resource files. For example, you may want to override a JDBC datasource XML module user name, password, and URL so that it references a local database.
```

Comment:
I think Weblogic domain configuration is more appropriate than Weblogic domain home configuration.

Reply:
The idea is to keep using 'domain home' when referencing a domain home configuration. The word   'domain' alone is very ambiguous, since the Operator project uses domain do describe the Domain kubernetes resource.